SmartDeviceLink/SDLUnsubscribeVehicleData.h

```diff
@@ -70,6 +70,7 @@ NS_ASSUME_NONNULL_BEGIN
  *  @param wiperStatus Unsubscribe to wiperStatus
  *  @return A SDLUnsubscribeVehicleData object
  */
+//(1)
```

Comment:
I'm not sure why "1", "2", and "3" have been added to the docs headers but we have to keep all of those inits since it would be a breaking change to remove them.